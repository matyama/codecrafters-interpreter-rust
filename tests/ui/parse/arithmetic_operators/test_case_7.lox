37 + 89 - 99
