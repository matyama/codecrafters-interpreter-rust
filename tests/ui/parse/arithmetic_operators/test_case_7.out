(- (+ 37.0 89.0) 99.0)
